src/components/done/__snapshots__/Done.test.tsx.snap

```diff
@@ -35,11 +35,19 @@ ShallowWrapper {
         />
         <p
             className="call__text"
-        />
-        <strong>
-            TOWN HALL COMPONENT HERE
-        </strong>
-        <br />
+        >
+            <Trans
```

Comment:
What prompted this change and including the `<0>` and `<1>` parts in the translation files? Feels like these things are getting tightly coupled with the extra required parameters